lib/src/base-styles/_nx-forms.scss

```diff
@@ -14,6 +14,12 @@
     color: var(--nx-swatch-red-40);
   }
 
```

Comment:
I'm wondering if `.nx-form` should be using the `container-vertical` mixin.  That would typically remove the top margin from the required field message (or whatever else is first) which is generally what we want.  See http://localhost:4043/#/pages/Container%20Helpers .  I'd definitely say try adding that, and if it breaks any visual tests outside of those intended for this PR, we'll figure out what to do from there.

Reply:
> I'm wondering if `.nx-form` should be using the `container-vertical` mixin. That would typically remove the top margin from the required field message (or whatever else is first) which is generally what we want. See http://localhost:4043/#/pages/Container%20Helpers . I'd definitely say try adding that, and if it breaks any visual tests outside of those intended for this PR, we'll figure out what to do from there.

Changed here: [5d3df01](https://github.com/sonatype/sonatype-react-shared-components/pull/735/commits/5d3df01ec33740d86c7ab63f215e25a2fbe3dea8)

After running the visual tests, the test failures came from changes in validation and removing the "optional" tag, which were all there previously. The only one I wanted to confirm is irrelevant is :

![nx-formlayout-js-nx-form-nx-fieldset-deprecated-layout-looks-right-1-diff](https://user-images.githubusercontent.com/89862727/178792962-32765f5b-ed25-4836-b633-09bec7dec53c.png)

The margin for the last child appears to be affected in this fieldset, but it seems to be coming from .nx-fieldset<:lastchild(2) vs .nx-fieldset<:last-child

![Screen Shot 2022-07-13 at 1 23 43 PM](https://user-images.githubusercontent.com/89862727/178793762-a98100aa-a06b-46f8-bf84-afb3d67c4e79.png)
<img width="510" alt="Screen Shot 2022-07-13 at 1 23 34 PM" src="https://user-images.githubusercontent.com/89862727/178793764-9e222d38-29a1-4669-9f6b-984b5d06b8cb.png">

Reply:
> After running the visual tests, the test failures came from changes in validation and removing the "optional" tag, which were all there previously. The only one I wanted to confirm is irrelevant is :

It's irrelevant to this PR yes.  It's a bug that needs to be fixed before the feature branch gets merged to main, and I think I addressed it in the RSC-1080 PR